### PR TITLE
Remove alternative 32_power realizations that are not used anymore 

### PR DESCRIPTION
**Pull request by lecfab**
## Purpose of this PR
In module 32_power, only IntC is used (see [conversation](https://mattermost.pik-potsdam.de/rd3/pl/yjs5pg8tujn3tcqgmkh6ouxuce) and [corresponding issue](https://github.com/remindmodel/development_issues/issues/320)).

- removed RLDC and DTcoup realizations
- renamed vm_capDistr vm_demSeOth vm_prodSeOth pm_boundCapCCS pm_cap0  according to their new scope
- updated mrremind, Katarzyna generated new input data: https://github.com/pik-piam/mrremind/pull/527
- updated remind2: 
  - https://github.com/pik-piam/remind2/pull/632
  - https://github.com/pik-piam/remind2/pull/637
  - https://github.com/pik-piam/remind2/pull/639
  - https://github.com/pik-piam/remind2/pull/644

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [x] Spring cleaning
- [x] Refactoring
- [ ] New feature 
- [x] Minor change (default scenarios show only small differences)
- [ ] Fundamental change
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [ ] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [x] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [ ] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [ ] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):

* Test runs are here:  `/p/tmp/fabricel/powerRealizations/output`
* Comparison of results (what changes by this PR?): nothing, just remove some realizations for 32_power

### CODE REVIEW

File: core/equations.gms
Comment (orichters) [inline core/equations.gms:441]: Why did you replace 15 by 2020-2015 = 5?

Comment (lecfab) [reply core/equations.gms:441]: Just to hide some bugs for more fun in the future 🤷 

It's a mistake thanks for digging and finding it!

**Review by orichters — APPROVED** [review]
It looks good from my side, although I still don't understand why this changes the `testOneRegi` behavior as noticed [here](https://github.com/pik-piam/remind2/pull/637). I would prefer if you could wait for @LaviniaBaumstark's approval before merging, maybe she has an idea.

Thanks for the clean-up.

**Comment by lecfab** [comment]
> I still don't understand why this changes the `testOneRegi` behavior as noticed [here](https://github.com/pik-piam/remind2/pull/637). I would prefer if you could wait for @LaviniaBaumstark's approval before merging, maybe she has an idea.

Yes let's do that!

**Comment by lecfab** [comment]
Now that we understand why the [issue with testOneRegi](https://github.com/pik-piam/remind2/pull/637#issuecomment-2304183928) occurred, I think we can merge this PR.
Further cleaning is ongoing in a separate issue about [removing demSeOth](https://github.com/remindmodel/development_issues/issues/338).

What do you think @orichters @fbenke-pik ?

**Comment by orichters** [comment]
Fine from my side.

File: core/equations.gms
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline core/equations.gms:164]: Is `!! *** RLDC removal` intended to conway any information?

Comment (lecfab) [reply core/equations.gms:164]: no, they are anchors for remaining todos (related to [this SeOth issue](https://github.com/remindmodel/development_issues/issues/338) with which you helped me today)

**Pull request merged**
